tests/performance-tests/generate_test_config.rb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+
+require 'json'
+require 'yaml'
+
+manifest_file = ENV.fetch("CF_MANIFEST")
+manifest = YAML.load_file(manifest_file)
+
+admin_password = manifest.fetch("properties").fetch("performance_tests").fetch("admin_password")
+api_url = manifest.fetch("properties").fetch("cc").fetch("srv_api_uri")
```

Comment:
Yes, it's possible to grab it from acceptance tests but in my opinion the fact that those values match is just coincidence, they don't have to. I would say it's better to separate performance tests properties as we may change them more flexibly in the future and add more. For instance right now we are hardcoding this https://github.com/alphagov/paas-cf/blob/117021589_request_performance_testing/tests/performance-tests/src/performance/load_test.go#L20 which is not perfect solution and should be retrieved from manifest. Unfortunately to do that we would have to extend this struct: https://github.com/cloudfoundry-incubator/cf-test-helpers/blob/master/helpers/config.go#L12 which means forking cf-test-helpers repo.